src/main/kotlin/org/wycliffeassociates/otter/jvm/persistence/database/daos/ResourceMetadataDao.kt

```diff
@@ -115,6 +115,29 @@ class ResourceMetadataDao(
                 }
     }
 
+    fun fetchByLanguageAndIdentifier(languageSlug: String, identifier: String, dsl: DSLContext = instanceDsl): ResourceMetadataEntity? {
+        return dsl
+                .select()
+                .from(DUBLIN_CORE_ENTITY.join(LANGUAGE_ENTITY)
+                        .on(DUBLIN_CORE_ENTITY.LANGUAGE_FK.eq(LANGUAGE_ENTITY.ID)))
+                .where(LANGUAGE_ENTITY.SLUG.eq(languageSlug))
+                .and(DUBLIN_CORE_ENTITY.IDENTIFIER.eq(identifier))
+                .fetchOne()
+                ?.let {
+                    RecordMappers.mapToResourceMetadataEntity(it)
+                }
+    }
+
+    fun fetchByIdentifier(identifier: String, dsl: DSLContext = instanceDsl): ResourceMetadataEntity {
```

Comment:
this isn't sufficient to retrieve a unique metadata entry; for example, all GLs will have a "ulb," so if you import the French ulb and the English ulb, you can't be sure that the fetchOne call will give you the one you want

Reply:
This function isn't being used anyway, I should've already removed it.